sycl/include/sycl/accessor.hpp

```diff
@@ -2009,13 +2009,17 @@ class __SYCL_EBO __SYCL_SPECIAL_CLASS __SYCL_TYPE(accessor) accessor :
 #ifndef __SYCL_DEVICE_ONLY__
     detail::constructorNotification(getMemoryObject(), impl.get(), AccessTarget,
                                     AccessMode, CodeLoc);
+    std::strncpy(padding, Other.padding, std::strlen(Other.padding));
```

Comment:
The padding is just to make sure the device-side and host-side classes have the same size, so you do not need to copy it as it should never actually be accessed. Same goes for swapping.